package.json

```diff
@@ -26,6 +26,8 @@
     "express": "^4.14.0",
     "lodash": "^4.17.4",
     "mongoose": "^4.7.8",
+    "passport": "^0.3.2",
```

Comment:
you're right, we are not using passport right now. I started to set it up, and then changed course.